xAPI-Communication.md

```diff
@@ -177,6 +177,8 @@ The lists above are not intended to be exhaustive. See requirements throughout t
 
 ### 1.3 Alternate Request Syntax
 
+__Note__: Unless supporting IE9 or lower, this section is not necessary for implementations of xAPI.
```

Comment:
This is not true. LRSs *MUST* support this and clients *MAY* use this even when not using IE9 or lower.